code/go/0chain.net/smartcontract/dbs/event/user_aggregate.go

```diff
@@ -206,41 +206,77 @@ func (edb *EventDb) updateUserAggregates(e *blockEvents) error {
 		}
 	}
 
-	ids := make(map[string]interface{})
-	for i := range updatedAggrs {
-		updatedAggrs[i].Round = e.round
-		ids[updatedAggrs[i].UserID] = struct{}{}
+	idSet := make(map[string]bool)
```

Comment:
interface should be preferable https://dave.cheney.net/2014/03/25/the-empty-struct

Reply:
Very insightful 💡Will apply